checker/src/main/java/org/checkerframework/checker/calledmethods/GenerateDataKey.astub

```diff
@@ -0,0 +1,8 @@
+package com.amazonaws.services.kms;
+
+import org.checkerframework.checker.calledmethods.qual.*;
+
+interface AWSKMS {
+     // The predicate implements a modified one-hot detector that permits e.g. setNumberOfBytes and withNumberOfBytes to be called together, but not setNumberOfBytes and withKeySpec. Basically, this predicate enforces that only one of the two categories are represented.
```

Comment:
What is "modified" about this?  It looks standard to me.
What are the "two categories"?  That seems like the key information that should be in the comment but isn't.
Nit: "one ... is represented", not "one ... are represented".

Reply:
I don't know. I think Martin Schäf wrote this comment. I'll rewrite it to be more clear what the predicate represents.